src/layout-pack.js

```diff
@@ -57,6 +60,26 @@ var PackLayout = React.createClass({
             children[i].style.top = min + 'px';
 
             columns[minIndex] = min + children[i].offsetHeight + margin;
+
+            var max = 0;
+            for (var d = 0; d < columns.length; d++) {
+                if (columns[d] > max) {
+                    max = columns[d];
+                }
+            }
+        }
+
+        this.el = this.refs.container;
+        // Old versions of React doesn't return the raw DOM node
+        if (!(this.el instanceof window.Node)) {
+            this.el = this.el.getDOMNode();
+        }
+
+        var calcTotalHeight = max + 'px';
+        this.el.style.height = calcTotalHeight;
+
+        if (typeof this.props.onReposition === 'function') {
```

Comment:
Since the propTypes already check the type, can we just do `if (this.props.onReposition)` instead?